# input topic name setting ===========> Please change according to your dataset
lidar_topic: "/odom_lidar"
pose_source: 2 # [0:tf_tree, 1:tf_topic, 2:odom_topic]
odom_topic: "/auto_odom"

world_frame: "map"
child_frame: "lidar"

debug_print: true

# Ignore points closer than this value (meters) (default 0.4)
min_scan_range: 2.0

# Ignore points far than this value (meters) (default 12.0)
max_scan_range: 70.0

# Check on paper
sdf_space_carving: true

sdf_trunc: 0.3

# voxel resolution (meters)
sdf_voxel_size: 0.1

# min weight setting TODO add details description
sdf_min_weight: 0.5
sdf_deactivate: 0.05

# Translation
tx: 0
ty: 0
tz: 0
# Rotation Quaternion
x: 0
y: 0
z: 0
w: 1
